cornell-dti/office-hours
# Added TA and student comments to discussion questions

### Summary 
Added TA and student comments to discussion questions, and made slight style changes on the discussion question component.



I implemented TA and student comments so both TAs and students can comment on discussion questions, which other people can also see. I also made some slight style changes on the upvote button. I made slight changes in the types for FireQuestion to give the capability of comments to discussion questions.





### Test Plan 
- Add a comment as a student on a discussion question
- Add a comment as a TA on a discussion question
- See if you can see the student comment as a TA, and vice versa


### Notes 



### Breaking Changes 

None


I updated existing types in `/src/components/types/`
